src/NovelImageRect.cpp

```diff
@@ -9,16 +9,33 @@
 #include "GeoBounds.h"
 
 namespace NovelRT {
-
 NovelImageRect::NovelImageRect(NovelLayeringService* layeringService,
                                const float& screenScale,
                                const GeoVector<float>& size,
                                const std::string_view imageDir,
                                const NovelCommonArgs& args,
                                const GLuint programId,
-                               const RGBAConfig& colourTint) :
-    NovelRenderObject(layeringService, screenScale, size, args, programId),
-    _imageDir(imageDir), _colourTint(colourTint) {
+                               const RGBAConfig& colourTint) : NovelRenderObject(layeringService,
+                                                                                 screenScale,
+                                                                                 size,
+                                                                                 args,
+                                                                                 programId),
+                                                               _imageDir(imageDir), _colourTint(colourTint),
+                                                               _uvBuffer(Lazy<GLuint>([] {
```

Comment:
Since this pattern is repeated and so small, would it be a good idea to add `protected: static NovelRenderObject::createGenBuffer() const`?

These all then become `_uvBuffer(Lazy<GLuint>(createGenBuffer))`

Reply:
Fixed!